.github/workflows/ci.yml

```diff
@@ -29,7 +29,14 @@ jobs:
               automake \
               ca-certificates \
               git \
+              libhttp-daemon-perl \
+              libhttp-daemon-ssl-perl \
```

Comment:
``libhttp-daemon-ssl-perl`` requires ``libhttp-daemon-perl`` already so the later can be removed.